ui/mods/msu/mod_settings/settings_screen.js

```diff
@@ -233,6 +233,11 @@ ModSettingsScreen.prototype.switchToPage = function (_mod, _page)
 			while (setting.title.width() >= 341)
 			{
 				setting.title.css('font-size', (parseInt(setting.title.css('font-size').slice(0, -2)) - 1) + 'px');
+				if (parseInt(setting.title.css('font-size').slice(0, -2)) <= 1)
+				{
+					console.error("Font size of title too small! Stopping font size adjustment.");
```

Comment:
Maybe print the id (or at least the title if it's not available) to ease debugging just in case